### ISSUE TITLE
Make chrome notifications show bigger icon and the sending device in the subtext

### ISSUE THREAD

**Issue by joaomgcd** [joaomgcd/JoinChrome#20]
Kinda like this: http://imgur.com/bViem9B

**Comment by tommyent** [comment]
I think this is a matter or larger images from Android right? From what I see the images are not big enough. Same for the contextual commands. If not let me know what I can do

**Comment by joaomgcd** [comment]
I was thinking we can resize the icons a bit in javascript before using them in the notification. But I'll need to check out how to do that :)

**Comment by tommyent** [comment]
Is this to save on bandwidth? Sorry I just don't know how the notifications work. I'll try and look into it tonight

**Comment by joaomgcd** [comment]
Yeah, it is. Got to keep those icons as small as possible. Thanks in advance!

**Comment by tommyent** [comment]
@joaomgcd I never got a chance to look into this but I'm going to try over the weekend. What's currently using GCM?
I noticed they are base64 and if I remember correctly I remember seeing some base64 code or something in the extension repo. Is this being done on the server or extension? Just curious about the process from phone to end device and how it works. 
Also I assume bandwidth is the issue and not storage and theres no cost getting the image to your server from the device it's all egress right?

**Comment by joaomgcd** [comment]
The icons are currently sent using GCM directly :) Their content comes through encoded as Base 64 in the GCM payload itself, so most of them don't even go through my server at all. But since 1 GCM message can only contain 4kB, I shouldn't make them better quality anyway because I don't want to need to send multiple GCM messages per notification.

Notifications for windows 10 devices still go through my server though, so there's that.

Hope this clears things up! Thanks again
